tools/stress/src/org/apache/cassandra/stress/StressAction.java

```diff
@@ -155,15 +159,22 @@ private boolean runMulti(boolean auto, UniformRateLimiter rateLimiter)
             results.add(result);
 
             if (prevThreadCount > 0)
-                System.out.println(String.format("Improvement over %d threadCount: %.0f%%",
-                        prevThreadCount, 100 * averageImprovement(results, 1)));
+            {
+                double avgImprovement = 100 * target.averageImprovement(results, 1);
```

Comment:
for latency, the "improvement" is hard to describe as the test is stressing a cluster in such a way that even throughput is better, the latency will suffer so it will be progressively worse. The default returned from `averageImprovement` is `0` if not overloaded in the implementations.